app/src/main/java/app/revenge/manager/ui/activity/MainActivity.kt

```diff
@@ -40,14 +31,6 @@ class MainActivity : ComponentActivity() {
             )
         }
 
-        if (preferences.installMethod == InstallMethod.SHIZUKU) {
-            lifecycleScope.launch {
-                if (!ShizukuPermissions.waitShizukuPermissions()) {
-                    preferences.installMethod = InstallMethod.DEFAULT
-                }
-            }
-        }
```

Comment:
Why was this removed?

Reply:
It's no longer requested on app launch, only at usages of the installer